dist/after_msg.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="pl">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1, user-scalable=no"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta name="title" content="Patrycja Kołaczyńska - architektura" />
+    <meta name="description" content="Patrycja Kołaczyńska - architektura" />
+    <title>Patrycja Kołaczyńska - architektura</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&family=Rozha+One&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/main.min.css" />
```

Comment:
to powinno być pod slick.css, a to dlatego, że czasem trzeba nadpisać style i żeby je nadpisać to powinna być zachowana kolejność